dist/cjs/index.cjs

```diff
@@ -0,0 +1,513 @@
+'use strict';
```

Comment:
Does this need to be checked into git?  If the package is going to be used from npm then it doesn't need to be.

Reply:
At the moment it does actually as all installs of the reporter use github. If we publish in the future we should delete it. I'll make an issue for this.

Reply:
https://github.com/digitalbazaar/mocha-w3c-interop-reporter/issues/11

Reply:
I figured that's why it was added.  Not sure what the timeline is on this, but it would be nicer if merging generated code into main could be avoided.  Maybe use a branch with that on it and remove the branch later.  Do what's needed though.

Reply:
@davidlehn go ahread and release this I will make a branch in the interum with the build product.